src/client/multitransport-client.ts

```diff
@@ -51,7 +52,10 @@ export interface RequestOptions {
    */
   signal?: AbortSignal;
 
-  // TODO: propagate extensions
+  /**
+   * A key-value map for passing horizontally applicable context or parameters.
```

Comment:
Let's mention that all parameters are passed to the server via underlying transports. Similar to [this comment](https://github.com/a2aproject/a2a-go/blob/a08b9a84cc00e89932211f014732340214000e2a/a2aclient/middleware.go#L26-L29).